src/apis/fs.cpp

```diff
@@ -420,89 +438,90 @@ static int fs_open(lua_State *L) {
             lua_createtable(L, 0, 4);
```

Comment:
It may be valuable to deduplicate this code, as both virtual and real mounts now use the same function set.